de.metas.business/src/main/java/de/metas/payment/api/impl/PaymentDAO.java

```diff
@@ -23,15 +23,23 @@
  */
 
 import java.math.BigDecimal;
+import java.sql.PreparedStatement;
```

Comment:
i would ask you to clean up this imports, but i guess they will be needed when the updateDiscountAndPayment implementation is back :-)